src/components/v5/common/WidgetCards/partials/TitleComponent.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { type FC } from 'react';
+
+import { titleClassName } from '../styles.ts';
+
+interface TitleProps {
+  title: React.ReactNode;
+}
+
+export const TitleComponent: FC<TitleProps> = ({ title }) => (
+  <span>
+    <h3 className={titleClassName}>{title}</h3>
+  </span>
```

Comment:
Is there an advantage to nesting `h3` within `span` here?

Reply:
Thanks for noticing that, regarding mobile position I will handle it as part of #2851 , thanks!